### ISSUE TITLE
UX | AuthMenu Claims and Disability Rating | Auth menu mock-ups / IA discovery

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#72192]
## Background

In #71332 we decided to add the claim status tool to the auth menu, and are strongly considering adding the disability rating tool as well.  We need to mock this up and consult with CAIA.

The OCTO leads have determined that the team working on the new header/footer will work on adding these links to those designs, so this ticket will just focus on the current menu design.

## Tasks

- [x] Mock up the current menu with the new link additions
- [x] Consult with CAIA, specifically to discuss thoughts around chunking links. At what point do we need to do this? Also discuss:
- [x] Consider: after adding more links, at what point do we do usability testing/user research? When do we think testing of this menu is warranted?

## AC

- [x] We know how we want to proceed with regard to research
~- [ ] We're ready for FE implementation~

**Comment by andaleliz** [comment]
@Samara-Strauss I'm hoping I'll be able to complete this ticket before the end of the year, since I'll be on PTO 1/2-1/8.  However, if CAIA folks are out next week, it may not be completed until the 2nd or 3rd week of January. I'm not sure if you have a time frame in mind for this work, but wanted to give you a heads up.

**Comment by Samara-Strauss** [comment]
@andaleliz totally fine, and thanks for letting me know. Let's slot this into sprint planning accordingly cc/ @mtcA6

**Comment by mtcA6** [comment]
Missed this, I've tagged it with the refinement label so we can talk about it w/the team. I don't think everyone knows we're considering another round of expansions to the menu.

**Comment by andaleliz** [comment]
Adding notes from @mtcA6:

Samara discussed the header/footer work w/Dave C Thursday afternoon and then she reached out to me to clue me in on the future of the Auth Menu as it relates to the redesigned site. In discussing the auth menu w/the other OCTO folks they mentioned there's a round of research coming up for the redesigned header and footer (i think its round 2 of research). They thought it would be good if our team would run a round of usability testing for the auth menu that coincides with this second round of research. 
- I believe Samara gave the sitewide team the list of items that is already in the menu + the two additional items you recommended last sprint and they'll give us the designs to use for the research. 
- I believe they're standing up some codepen pages for their research and we'd be able to use that for research on the menu. 

Samara mentioned that we wouldn't work directly on the design of the auth menu in the redesigned header/footer but would focus more narrowly on the items included in the menu and the usability of the design supplied by the OFCIO. 
  - Based on the study/participant feedback, I think this necessarily would give us (you/pam) some say/input into the design choices for the auth menu. Any changes/recommendations to the auth menu would, as I understood it, flow back to the design system for the broader OFCIO updates. 

Our main goal is to, once OFCIO creates the auth menu in their designs:
- run usability testing
- provide a topline summary ASAP, no later than March
- Samara said a more detailed report could be completed after the end of March deadline if we felt it was necessary, but the topline summary should be completed before then.

**Comment by andaleliz** [comment]
Travis and I met today to discuss, and we have a few questions that will be helpful have answered as we plan research:

- Is the design team going to consult with CAIA at all on IA/chunking or not to chunk? Or is there an IA person involved in the work?
- We need to see their prototype so we can figure out the scope of the tasks we can ask people to go through
- Since the ask is specifically usability testing, are there any specific questions we know we want answered at this point?
  - I have some IA questions and was considering whether or not a card sort would be better, but I think we can answer those in a usability test as well as any other questions we need answered. A moderated card sort + usability combo study could be interesting to explore too.

**Comment by mtcA6** [comment]
Created a [first draft of the project outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/authenticated-menu/add-claims-and-disability-rating/README.md). I kept the KPIs that we had from Letters and Dependents work we did earlier this year. I was going to remove the profile specific KPI's and then decided to keep them, but since this isn't specific to the profile I think we could safely remove the metrics focused on searches from within the profile. 

What we tracked for Letters and Dependents would definitely hit on the first bullet in [Liz's comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71332#issuecomment-1847858832), but the the interactions with the tools might be slightly harder to get at. 

>If we move forward with the experiment, I think we could track a couple of things as success metrics:
>- engagement w/ the nav link
>- interactions within the tools, somehow. Something to show that any increase in traffic is parallel to increase with tool engagement. I don’t think we’d want to see more traffic and pageviews without engagement also going up.

For the interactions, I think we might need to create some funnel charts in GA, I can attend an analytics office hours section in the new year to ask about this. I know we have tracking on the actual menu clicks but we do not [have tracking on the opening/closing of the menu because of the GA4 migration](https://github.com/department-of-veterans-affairs/va.gov-team/issues/66653).

**Comment by andaleliz** [comment]
@mtcA6 That doc is a great start! Thank you for putting it together.

>since this isn't specific to the profile I think we could safely remove the metrics focused on searches from within the profile.

I think that's the right call, and I also think we can remove the reference to profile in the undesired user outcomes section. IIRC, people aren't looking for these things in the profile (though maybe I'm forgetting something).

>the interactions with the tools might be slightly harder to get at....For the interactions, I think we might need to create some funnel charts in GA, I can attend an analytics office hours section in the new year to ask about this.

It's a great idea to go to office hours and see what they can suggest or help with.  Would it be possible to collaborate with the claim status tool and view disability rating tool teams to get their input ? Maybe they track something already that would make sense to monitor once we add these links.

I spent a chunk of the day thinking about this and also consulted with a senior IA to discuss my approach and questions on research. There isn't a widely agreed-upon best practice in terms of how many items you can have in a nav item before you have to chunk, do research, etc. We talked about [Miller's Law ](https://en.wikipedia.org/wiki/The_Magical_Number_Seven,_Plus_or_Minus_Two), which feels like a good guardrail to me. This also lines up with my gut instinct that we're OK adding these two items to the menu and learning from analytics like we did previously.  I'd like to keep them as one group for now, to avoid introducing multiple variables into the experiment.

I'm less certain about how to order the items.  There are two options I'm considering.  The first ([mock-up](https://www.sketch.com/s/b1dacba9-064d-4a24-bfa4-ae0da5790f08/a/pY5KMjR)) is to keep the broader personalized tools together at the top, and then group the single-purpose tool links in order of usage/popularity of search terms:
My VA
My HealtheVet
Profile
Claim status
Disability rating
Dependents
Letters

The second option ([mock-up](https://www.sketch.com/s/b1dacba9-064d-4a24-bfa4-ae0da5790f08/a/YZ85xRO)) is to prioritize benefit specific links by importance to veterans (which I think reflects usage/popularity, though we have less data about health tools since many are on MHV), and put broader tools at the bottom.

My healtheVet
Claim status
Disability rating
Dependents
Letters
My VA
Profile

Ideally, I'd love to A/B test both (this is what the IA recommended) but I know that's a little tricky on VA.gov. I also considered Alphabetical order, and Maintaining the current order and adding the new links at the bottom of the list. 

@Samara-Strauss what are your thoughts on this? Even if we decide not to run this experiment on the current design, we can provide an order recommendation to the sitewide team and test this in the study we run.

**Comment by andaleliz** [comment]
@mtcA6 and I had a convo earlier and he asked a good question about whether or not I had considered putting profile first. "Your account" or "profile" are the top 1-2 links on a lot of bigger sites like Amazon, Target, etc.  

I had thought about it, and the reason I didn't land on that is really because I don't think profile related tasks are more important to veterans than things they can do on My VA or the other tools we're linking to. But, I def see the logic behind profile being the first item in a user menu, and am totally open to trying that if it makes more sense from a product perspective.

**Comment by Samara-Strauss** [comment]
Hi all! Happy New Year!

> Does this mean we don't want to add these to the current menu in the interim

We can discuss this, but since we'll be testing these in a prototype, I think we can hold off on adding these to the menu and wait to make these updates until after testing. I think an argument could be made for not waiting, so let me know if you feel strongly about a different approach.

> Is the design team going to consult with CAIA at all on IA/chunking or not to chunk?

I don't believe so. It sounds like they really want us to handle everything with the auth menu.

> Since the ask is specifically usability testing, are there any specific questions we know we want answered at this point?

We should focus on usability testing, but we could also take advantage of having people in front of us and ask questions around their expectations of this menu. I'm also especially interested in how we can differentiate this as a personalized menu on mobile, since this menu will not now be grouped with the general/un-personalized menus.

> I have some IA questions and was considering whether or not a card sort would be better, but I think we can answer those in a usability test as well as any other questions we need answered. A moderated card sort + usability combo study could be interesting to explore too.

Let's talk about additional research we want to run around the auth menu, and when it would be appropriate to do that. I do think a card sort -- or a series of card sorts -- could be useful to get a broader view of auth navigation in general.

> Created a [first draft of the project outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/authenticated-menu/add-claims-and-disability-rating/README.md). I kept the KPIs that we had from Letters and Dependents work we did earlier this year. I was going to remove the profile specific KPI's and then decided to keep them, but since this isn't specific to the profile I think we could safely remove the metrics focused on searches from within the profile.

@mtcA6 I am removing the KPIs around reducing Claims and Disability Rating searches, as [neither of these are in the top searches for the profile or website over the last month](https://va-gov.domo.com/page/1834995012?userId=66061986). I'll think about what metrics we can use to measure behavior change.

> I'm less certain about how to order the items. 

I commented on this in Slack as well, but this is my preferred order.

> My VA
> My HealtheVet
> Profile
> Claim status
> Disability rating
> Dependents
> Letters
